testing/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should need a package.json file here. The package.json file in the root is where all test dependencies are.

Reply:
Let me know what failed without this. The folder is just meant for testing utilities.

Reply:
I changed a data type from string to TextDocument in the tests, so the tests need `vscode-languageserver`.

Reply:
That makes sense. But then `vscode-languageserver` should be added to the root package.json file as a root dependency. That should work. 🤔

Reply:
But `vscode-languageserver ` is not installed as a devDependency in the root package.json - that is why it fails. Can you try doing that instead of adding it to another package.json file inside the testing folder. 

Basically we have a package containing all infrastructure needed to type, lint and test in the root package.json. This should contain additional dependencies needed for testing. 

Thanks. :)

Reply:
This still fails

```diff
diff --git a/package.json b/package.json
index 8680641..c0330f1 100644
--- a/package.json
+++ b/package.json
@@ -24,7 +24,8 @@
     "tslint-config-prettier": "^1.10.0",
     "tslint-plugin-prettier": "^1.3.0",
     "tslint": "^5.9.1",
-    "typescript": "^2.8.1"
+    "typescript": "^2.8.1",
+    "vscode-languageserver": "^4.1.1"
   },
   "jest": {
     "testEnvironment": "node",
diff --git a/testing/package.json b/testing/package.json
index aa082ff..9664f26 100644
--- a/testing/package.json
+++ b/testing/package.json
@@ -1,5 +1,4 @@
 {
   "dependencies": {
-    "vscode-languageserver": "^4.1.1"
   }
 }
```

Can you make the change you want instead of telling me?

Reply:
Ugh, I didn't run `yarn install` before `yarn run check:bail`, it works after all.